kernelci/cli/base.py

```diff
@@ -533,6 +537,9 @@ def __init__(self, path, command, cli_args, section=None):
                   way to have default values for each CLI tool
         """
         self._deprecated_settings = False
+        if path is None and os.environ.get('KCI_SETTINGS'):
```

Comment:
Alright I think this should tick all the boxes, and the rest can be left unchanged:
```suggestion
        if path is None:
            env_path = os.environ.get('KCI_SETTINGS')
            if env_path:
                if not os.path.exists(env_path):
                    raise FileNotFoundError(env_path)
                path = env_path
```

Does that make sense?  Sorry for all the ping-pong, maybe it would have been better to just use `--settings` rather than add support for `KCI_SETTINGS`.  Now we have it so I think that's fine.

Reply:
I think environment variable handy in some cases, especially docker